File: 24_C#Fundamentals/_01_BasicSyntaxConditionalStatementsAndLoops/_01_Lecture/_06_ForeignLanguages/_06_ForeignLanguages.cs
using System;

public class _06_ForeignLanguages
{
    public static void Main(string[] args)
    {
        // Read input value
        string country = Console.ReadLine();

        // Print country based on the input
        switch (country) 
        {
            case "England":
            case "USA":
                Console.WriteLine("English");
                break;
            case "Spain":
            case "Argentina":
            case "Mexico":
                Console.WriteLine("Spanish");
                break;
            default:
                Console.WriteLine("unknown");
                break;
        }
    }
}

File: 02_ProgrammingFundamentals/_01_BasicSyntaxConditionalStatementsAndLoops/_01_Lecture/_06_ForeignLanguages.java
package _02_ProgrammingFundamentals._01_BasicSyntaxConditionalStatementsAndLoops._01_Lecture;

import java.util.Scanner;

public class _06_ForeignLanguages {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input value
        String country = scan.nextLine();

        // Print country based on the input
        switch (country) {
            case "England":
            case "USA":
                System.out.println("English");
                break;
            case "Spain":
            case "Argentina":
            case "Mexico":
                System.out.println("Spanish");
                break;
            default:
                System.out.println("unknown");
        }

        scan.close();
    }
}
